### PR TITLE
update readme 

### PR DESCRIPTION
**Pull request by aylacura**
some additions to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,65 @@
...
+
+## Contributions
+
+Contributions are welcome!
+
+    1. Fork the repo
+    2. Create a new branch `git checkout -b improve-feature`
+    3. Make the appropriate changes in the files
+    4. Add changes to reflect the changes made 
+    5. Commit your changes `git commit -am 'Improve feature'`
```
Comment (JeppeHauman) [inline README.md:59]: Typo

File: README.md [hunk 2]
```diff
@@ -1,18 +1,65 @@
...
+    6. Push to the branch `git push origin improve-feature`
+    7. Create a Pull Request
 
 ## License
 
-[MIT](https://choosealicense.com/licenses/mit/)
+[MIT](https://choosealicense.com/licenses/mit/) &copy; Lagsi
```
Comment (JeppeHauman) [inline README.md:65]: I like it!

**Review by JeppeHauman — COMMENTED** [review]
Is this how it works?

File: README.md (hunk 1 above)
Comment (aylacura) [reply README.md:59]: Apparently you can stage and commit with that command. I just copied it from somewhere. Should I change?

Comment (n13a) [reply README.md:59]: yeah It's a concise way of doing this:

`git add .`
`git commit -m "commit message"`

Comment (JeppeHauman) [reply README.md:59]: oooooooohhhh leave it then

**Pull request merged**